nailgun/entities.py

```diff
@@ -2912,6 +2912,42 @@ def __init__(self, server_config=None, **kwargs):
             'to': entity_fields.IPAddressField(null=True),
             'vlanid': entity_fields.StringField(null=True),
         }
+        if (getattr(server_config, 'version', parse_version('6.1')) >=
+                parse_version('6.1')):
+            self._fields['boot_mode'] = entity_fields.StringField(
+                choices=('Static', 'DHCP',),
+                default=u'DHCP',
+                null=True,
+            )
+            self._fields['dhcp'] = entity_fields.OneToOneField(
+                SmartProxy,
+                null=True,
+            )
+            # self._fields['discovery'] = entity_fields.OneToOneField(
```

Comment:
Can you add a comment saying why this is commented out? Bonus points for creating an issue and linking to it. #52 may serve as a template for you.